libcamera/src/camera_manager.rs

```diff
@@ -31,10 +31,17 @@ impl CameraManager {
     }
 
     /// Enumerates cameras within the system.
-    pub fn cameras(&self) -> CameraList<'_> {
+    pub fn cameras<'a>(&self) -> CameraList<'a> {
         unsafe { CameraList::from_ptr(NonNull::new(libcamera_camera_manager_cameras(self.ptr.as_ptr())).unwrap()) }
     }
 
+    pub fn stop(&self) {
+        unsafe {
+            libcamera_camera_manager_stop(self.ptr.as_ptr());
+            // libcamera_camera_manager_destroy(self.ptr.as_ptr());
```

Comment:
This commented out code implies that your not sure if this should or should not be called?

@kbingham do you have any insite on this one?

Reply:
this allows for some one to create a camera

call stop()

have the camera be stopped and then when tha camera is dropped

Stop will be called again, followed by destroy.

I assume this is ok?

Reply:
I don't think this is safe. This is what libcamera docs say:
```
/**
 * \brief Stop the camera manager
 *
 * Before stopping the camera manager the caller is responsible for making
 * sure all cameras provided by the manager are returned to the manager.
 *
 * After the manager has been stopped no resource provided by the camera
 * manager should be consider valid or functional even if they for one
 * reason or another have yet to be deleted.
 */
void CameraManager::stop()
```

So I will remove it in #68. The unplugged camera case needs to be investigated separately.